### DISCUSSION TITLE
Shape of 1-dimensional arrays

### DISCUSSION THREAD

**Discussion by catkira** [catkira/py3gpp#42] [General]
In Matlab all arrays that should be 1-dimensional are two dimensional. The last axis just has length of 1 in that case.
How should we handle this in py3gpp? Always add this empty dimension or not?

@vborchsh

**Comment by vborchsh** [comment]
This feature is the MATLAB only feature because it has advanced matrix/vector engine. So, we can go Python-way, I think. There is no reason to keep _everything_ the same.

**Reply by catkira** [reply]
yes, I agree!
